docs/_running-tests/chrome.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Chrome
+---
+When running Chrome, there are some additional useful command line arguments.
+
+As with most products, you can use a different binary with `--binary`, e.g., to
+run Chrome Dev on Linux:
+
+```
+./wpt run --binary `which google-chrome-unstable` chrome
+```
+
+Extra arguments to Chrome can be passed with `--binary-args`.
+
+To enable experimental web platform features:
+
+```
+./wpt run --binary-arg=--enable-experimental-web-platform-features chrome fullscreen/
```

Comment:
nit: can you add a link that gives more explanation of what this flag does?

Reply:
Will link to https://www.chromium.org/blink/runtime-enabled-features and chrome://flags/#enable-experimental-web-platform-features

Reply:
Well, that was harder than I thought, but PTAL.